computer-use-demo/computer_use_demo/streamlit.py

```diff
@@ -224,7 +224,7 @@ def _reset_api_provider():
                     tab=http_logs,
                     response_state=st.session_state.responses,
                 ),
-                api_key=st.session_state.api_key,
+                api_key=st.session_state.api_key.replace(" ", ""),
```

Comment:
preference for https://docs.python.org/3.11/library/stdtypes.html#str.strip since it generically removes whitespace at the start/end
```suggestion
                api_key=st.session_state.api_key.strip(),
```